spec/models/manageiq/providers/vmware/infra_manager/metric_capture_spec.rb

```diff
@@ -0,0 +1,136 @@
+require Rails.root.join('spec/tools/vim_data/vim_data_test_helper')
```

Comment:
jfyi, if this is run in a plugin you'll want to use relative paths.
because `Rails.root` is the root of the dummy app then.